src/index.js

```diff
@@ -1,5 +1,29 @@
 import './style.css';
-import addScore from './functions';
+import {
+  addNewGame, getScore, addScore, postScore,
+} from './functions';
 
-const btn = document.querySelector('.submit-button');
-btn.addEventListener('click', addScore);
\ No newline at end of file
+async function consumeAPI() {
+  const gameData = await addNewGame('games/', 'java game');
+  const id = gameData.result.split(':')[1].trim().split(' ')[0];
+  const refresh = document.querySelector('.refresh');
+  const submitBtn = document.querySelector('.submit-button');
+  refresh.addEventListener('click', () => {
```

Comment:
- [ ] <b>REQUIRED:</b> When `refresh` button is click, it is duplicating the previous data. It should not do that. Kindly fix it.
![scrnli_11_10_2021_10-15-16 PM](https://user-images.githubusercontent.com/45217364/141186878-cf473b84-5e04-4d62-a7bd-6323c37f4a38.png)